widget/popup_test.go

```diff
@@ -3,10 +3,12 @@ package widget
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	"fyne.io/fyne"
+	"fyne.io/fyne/internal/cache"
 	"fyne.io/fyne/test"
 	"fyne.io/fyne/theme"
-	"github.com/stretchr/testify/assert"
```

Comment:
I think we order with github after fyne. I'm surprised goimports did this...

Reply:
GoLand did this. It imports local packages after remote after system.

Reply:
Have you told it to use goimports sort order?

Reply:
![grafik](https://user-images.githubusercontent.com/73959/75219630-98027000-579d-11ea-8ae9-a1cf94dca314.png)

Honestly, I like the idea of separating external and internal imports.

Reply:
Me too :) but github is external and fyne is internal, so surely the previous order is correct (a space probably belonged in there between them though)

Reply:
Okay, I resorted it.